src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -286,10 +316,9 @@ describe("Offer InitialMutation", () => {
 
     it("tracks the offer input focus", () => {
       expect(mockPostEvent).not.toHaveBeenCalled()
-
+      page.selectCustomAmount()
```

Comment:
From now on the OfferInput is not visible until the user clicks on the custom option. So this first step is needed.